department-of-veterans-affairs/caseflow
# Remove status field from DB and instead calculate from conference job status

Resolves #[13778](https://github.com/department-of-veterans-affairs/caseflow/issues/13778)

### Description
Replaces the status column in the database with a calculated status based on whether the Pexip conference has been created, cancelled, or is in the process of being created.

### Acceptance Criteria
- [ ] Remove `status` column from virtual hearings table
- [ ] Ensure we are determining the status of a virtual hearing in a consistent way across the application
- [ ] Ensure functionality hasn't changed

### Testing Plan
1. Select judge "BVAYELLOW": http://localhost:3000/test/users
1. Navigate to hearings:  http://localhost:3000/hearings/schedule
1. Choose a "Video" hearing
1. Click the "Edit hearing details" link
1. Change from a "Video" hearing to a "Virtual" hearing and ensure you get a success message
1. Navigate back to hearings and confirm the hearing is now "Virtual": http://localhost:3000/hearings/schedule
1. Navigate back to the previously selected hearing and change it back to "Video"
1. Once again navigate to hearings and confirm the hearing is back to "Video": http://localhost:3000/hearings/schedule

### Database Changes

* [x] DB schema docs updated with `make docs`
